oci/tests/BUILD.bazel

```diff
@@ -35,36 +31,17 @@ platform(
         ],
         visibility = ["//visibility:public"],
     )
-    for image, digest in {
-        "java17": "sha256:161a1d97d592b3f1919801578c3a47c8e932071168a96267698f4b669c24c76d",
-        "static": "sha256:c3c3d0230d487c0ad3a0d87ad03ee02ea2ff0b3dcce91ca06a1019e07de05f12",
-    }.items()
+    for repo_name, reference in IMAGES_TO_TEST.items()
 ]
 
-platform_transition_filegroup(
-    name = "distroless_java_amd64",
-    # This one is declared with an oci_pull rule in /WORKSPACE
-    srcs = ["@distroless_java"],
-    target_platform = _PLATFORM,
-)
-
-diff_test(
-    name = "test_java17",
-    file1 = "java17",
-    file2 = "distroless_java_amd64",
-)
-
-platform_transition_filegroup(
```

Comment:
how come we needed the transition before and don't need it now?

Reply:
this was added to get past the select statement we have in oci_alias and get the concrete oci_pull which would `amd64` that is equivalent to `{name}_linux_amd64`.